test/Grpc.Net.Client.Tests/Infrastructure/GrpcWebHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+namespace Grpc.Net.Client.Web;
+
+public class GrpcWebHandler : DelegatingHandler
```

Comment:
This is sketchy, why not use the already existing public GrpcWebHandler?

Reply:
```suggestion
// Use a GrpcWebHandler type here with the same name and namespace because Grpc.Net.Client.Tests doesn't reference Grpc.Net.Client.Web.
public class GrpcWebHandler : DelegatingHandler
```

Reply:
I added a comment explaining the reason.

Reply:
And why not add a reference to Grpc.Net.Client.Web?

Reply:
I tried that, and it worked 😅 

If you're wondering why I did this, it's because Grpc.Net.Client.Web.Test can't reference Grpc.Net.Client because of type conflicts. I eventually moved where I added this test to the other project, and didn't attempt the opposite. Turns out it is ok.